From 2295259b273ec4280e240f546888ef8d7cea414b Mon Sep 17 00:00:00 2001
From: Ivan Molodetskikh <yalterz@gmail.com>
Date: Mon, 7 Oct 2024 08:41:11 +0300
Subject: [PATCH] Set manual-lifetime for Tracy

Otherwise it tries to initialize its hardware timer and complains even
if we don't use it.
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 22ac37f..f14a3a3 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,4 +21,4 @@ tracing = "0.1.40"
 tracing-chrome = "0.7.2"
 tracing-log = "0.2.0"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
-tracing-tracy = "0.11.2"
+tracing-tracy = { version = "0.11.2", features = ["manual-lifetime"] }
-- 
2.46.2

